src/editor/EditorCommandHandlers.js

```diff
@@ -45,7 +43,7 @@ define(function (require, exports, module) {
     var commandKey = {
         COMMAND_KEY: "Ctrl"
     };
-    if(true){
+    if(navigator.platform.indexOf("Mac") !== -1;){
         commandKey = {
             COMMAND_KEY: "&#8984"
```

Comment:
I see in the [Mustache docs](https://github.com/janl/mustache.js#variables) the following: "All variables are HTML-escaped by default. If you want to render unescaped HTML, use the triple mustache: {{{name}}}. You can also use & to unescape a variable."